### ISSUE TITLE
Object detection not working on a custom dataset

### ISSUE THREAD

**Issue by darsh10** [yhenon/pytorch-retinanet#208]
Dear All,

I am having trouble with the following, fairly trivial setup. I have a new task "thread detection". Where we wish to identify a thread/stitch on a needle. I have a single image for training a model on this dataset. The image has only one object annotated. I wish to train a model on this image and then visualize it.

I have set up the dataset as correctly as possible in the cocodataset format. During training, the loss certainly goes down, indicating convergence. However, on inference, on the same image, the model can't identify the object. What could be going on?

I am training the model for 500 epochs.

**Comment by lukasmattheuwsen** [comment]
I'm not really certain if it will be possible to train a model on a single training image. You just don't give the network enough examples to learn what you are looking for. Although the loss is decreasing you don't know what the network is looking at and "learning".
For custom datasets you can use the CSV dataset format as this is pretty easy to setup tbh and worked fine for my implementation.
